build.gradle.kts

```diff
@@ -207,14 +209,11 @@ allprojects {
         isFailOnError = false
     }
 
-    afterEvaluate {
-        // Note: plugins should override this
-        version = getWPILibVersion() ?: getVersionFromGitTag()
-        tasks.withType<Jar> {
-            manifest {
-                attributes["Implementation-Version"] = project.version as String
-                attributes["Built-Date"] = Instant.now().toString()
-            }
+    // Note: plugins should override this
+    tasks.withType<Jar> {
```

Comment:
This should be `tasks.withType<Jar>().configureEach { ...` now.

Reply:
Fixed